File: June-18/02-06-18/java/01-The-Great-Kian.java

import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int ar[] = new int[n];
        for(int i = 0; i < n; i++)
            ar[i] = input.nextInt();
        long a = 0, b = 0, c = 0;
        
        for(int i = 0; i < n; i += 3)
            a += ar[i];
        for(int i = 1; i < n; i += 3)
            b += ar[i];
        for(int i = 2; i < n; i += 3)
            c += ar[i];
        
        System.out.println(a + " " + b + " " + c);
        
    }
}

File: June-18/02-06-18/python/01-The-Great-Kian.py
n = int(input())
ar = [int(x) for x in input().split()]

a = 0
b = 0
c = 0

for i in range(0, n, 3):
    a += ar[i]

for i in range(1, n, 3):
    b += ar[i]

for i in range(2, n, 3):
    c += ar[i]

print(a,b,c)
